Mods/EpipEncounters_7d32cb52-1cfd-4526-9b84-db4867bf9356/Story/RawFiles/Lua/Epip/QuickInventory/Filters/Equipment.lua

```diff
@@ -408,22 +420,75 @@ QuickInventory.Hooks.IsItemVisible:Subscribe(function (ev)
     ev.Visible = visible
 end)
 
--- Sort equipment by rarity.
-QuickInventory.Hooks.SortItems:Subscribe(function (ev)
-    if QuickInventory:GetSettingValue(QuickInventory.Settings.ItemCategory) == "Equipment" then
-        local rarityA, rarityB = ev.ItemA.Stats.Rarity, ev.ItemB.Stats.Rarity
-        local scoreA, scoreB
+---@param item EclItem
+---@return integer
+function ScoreLevel(item)
```

Comment:
Global functions are usable across all scripts, they are a no-no in the codebase.
Instead of `local` for auxiliary functions I prefer to make "private" functions in the feature by prefixing them with `_`. There is a setting in the LLS extension that lets this behave like the `@private` annotation, making it not appear in auto-complete outside of class declaration - I do this for such auxiliary functions I do not consider part of the feature's API